rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -32,8 +32,10 @@ Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby(abi)
 %endif
 
-Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.6
+Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.8
 Requires: %{?scl_prefix}rubygem(dynflow) < 0.9.0
+Requires: %{?scl_prefix}rubygem(parse-cron) >= 0.1.4
```

Comment:
This also needs adding as a BuildRequires, since it's a runtime dependency and the plugin's loaded during build.